src/linters/stylelint/index.js

```diff
@@ -0,0 +1,111 @@
+const Module = require( 'module' );
+const path = require( 'path' );
+const moduleAlias = require( 'module-alias' );
```

Comment:
Oh forgot to say, these should usually be alphabetical by module name; doesn't overly matter though.

```suggestion
const Module = require( 'module' );
const moduleAlias = require( 'module-alias' );
const path = require( 'path' );
```